Makefile

```diff
@@ -10,10 +10,13 @@ CHECKER := awk '{ print } END { if (NR > 0) { exit 1 } }'
 BR_PKG := github.com/pingcap/br
 
 VERSION := v5.0.0-master
+release_version_regex := ^v5\.[0-9]\.[0-9].*$$
 release_branch_regex := ^release-[0-9]\.[0-9].*$$
 ifneq ($(shell git rev-parse --abbrev-ref HEAD | egrep $(release_branch_regex)),)
-	# If we are in release branch, use tag version.
-	VERSION := $(shell git describe --tags --dirty)
+	# If we are in release branch, try to use tag version.
+	ifneq ($(shell git describe --tags --dirty | egrep $(release_version_regex)),)
```

Comment:
:thinking: <code>git describe --tags --dirty <strong>--match 'v5.*'</strong></code>

Reply:
I prefer the current way, because `git describe --tags --dirty --match ` outputs error messages. 

```
$ git describe --tags --dirty --match "v5.*"   
fatal: No tags can describe '09e9943d74259a6166398ef83fb8ac516793f55f'.
Try --always, or create some tags.
```

Reply:
okay